src/contents.ts

```diff
@@ -353,6 +365,13 @@ class ContentsManager implements IContents.IManager {
   }
 
   /**
+   * Get a download url given a file path.
```

Comment:
We should clarify that the path is a posix file path, *not* URL-encoded. Perhaps in a `@param` listing?